theano/tensor/nnet/nnet.py

```diff
@@ -2445,7 +2815,7 @@ def elu(x, alpha=1):
         "Fast and Accurate Deep Network Learning by
         Exponential Linear Units (ELUs)" <http://arxiv.org/abs/1511.07289>`.
     """
-    return tensor.switch(x > 0, x, alpha * tensor.expm1(x))
+    return tensor.switch(x > 0, x, alpha * (tensor.exp(x) - 1))
```

Comment:
For small values of x, tensor.expm1(x) provides higher precision. So it's better not to remove this.